meta-xt-domx/recipes-core/dbus/dbus-cxx_0.10.0.bb

```diff
@@ -6,8 +6,10 @@ DEPENDS = "boost dbus libsigc++-2.0"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
 
-SRC_URI = "git://github.com/dbus-cxx/${PN}.git;nobranch=1;tag=${PV};protocol=https"
+SRC_URI = "git://github.com/dbus-cxx/${PN}.git;nobranch=1;protocol=https"
```

Comment:
I have removed `tag` to avoid possible bitbake's confusion with `SRCREV` and `tag` defined at the same time.